cmd/qat_plugin/qat_plugin.go

```diff
@@ -203,21 +199,35 @@ func isValidDpdkDeviceDriver(dpdkDriver string) bool {
 	}
 	return false
 }
-
+func isValidVfDeviceID(vfDevID string) bool {
+	switch vfDevID {
+	case "0442", "0443", "37c9", "19e3":
+		return true
+	}
+	return false
+}
 func (dp *devicePlugin) scan() (deviceplugin.DeviceTree, error) {
```

Comment:
Nitpick: I think it's a good practice to separate functions visually with one empty line.